### ISSUE TITLE
Accessibility issue with ui5-checkbox as the shadow DOM input element with type=checkbox is missing aria-label attribute

### ISSUE THREAD

**Issue by rajatmtalwar** [SAP/ui5-webcomponents#2265]
**Describe the bug**
Accessibility issue with ui5-checkbox as the shadow DOM input element with type=checkbox is missing aria-label attribute
Axe identifies this as a problem 
![image](https://user-images.githubusercontent.com/4739468/94463343-668dcf00-01da-11eb-90be-9b1658e45c9c.png)

**To reproduce**
Goto https://sap.github.io/ui5-webcomponents/playground/components/CheckBox/
check the shadow dom for any of the checkboxes

**Expected behavior**
ui5-checkbox label value should be associated to the shadow input dom element with type "checkbox"

**Screenshots**
![image](https://user-images.githubusercontent.com/4739468/94463222-39d9b780-01da-11eb-8732-eeb85ae787bc.png)

**Affected components** *(if known)*
ui5-checkbox

**Comment by vladitasev** [comment]
I think we should have the same solution as for ui5-button.
